portfolio/src/main/webapp/script.js

```diff
@@ -10,4 +10,15 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+const navigate = (pageid) => document.getElementById(pageid).scrollIntoView({behavior:"smooth",block:"start"});
```

Comment:
[pick one of the valid formatting styles for object literals](https://google.github.io/styleguide/jsguide.html#formatting-object-literals)

Reply:
Format updated with commit d091e22
Note: I'm a little unsure here. Referring to [4.2.2](https://google.github.io/styleguide/jsguide.html#formatting-object-literals), I am attempting to style my object literal as described in the second example. Is the missing part the space between the ',' and the 'block' attribute?